src/pages/docs/src/codegen/gen-overview-page.js

```diff
@@ -0,0 +1,77 @@
+const { readFileSync } = require('fs');
+const { basename, resolve } = require('path');
+const { getPluginTypes, getPluginImage, getPluginCategory } = require('./paths');
+const { generatePage } = require('./pages');
+
+module.exports = function() {
+
+  const categories = [];
+  const fragments = [];
+  const children = getPluginTypes().map(file => {
+    const name = basename(file).replace('.json', '');
+    const image = getPluginImage(name);
+    let dest = '';
+    let data = null;
+    let pluginCategory = '';
+
+    try {
+      dest = resolve(__dirname, '..', '..', '..', '..', 'plugins', name, 'package.json');
+      data = JSON.parse(readFileSync(dest, 'utf8'));
+    } catch(e) {
+      dest = resolve(__dirname, '..', '..', '..', '..', 'converters', name, 'package.json');
+      data = JSON.parse(readFileSync(dest, 'utf8'));
+    }
+
+    pluginCategory = getPluginCategory(data);
+    this.addDependency(dest, { includedInParent: true });
+    return {
+      category: pluginCategory.charAt(0).toUpperCase() + pluginCategory.slice(1),
+      content: `
+        <ImageCard
+          link="/plugins/${name}"
+          image={require('../../assets/${image}')}
+          description="${data.description}"
+          title="${data.name}"
+        />`
+    };
+  });
+
+  for (const child of children) {
+
+    const index = categories.indexOf(child.category);
+
+    if (index === -1) {
+      categories.push(child.category);
+      fragments.push({
+        category: child.category,
+        children: [child.content],
+      });
+    } else {
+      fragments[index].children.push(child.content);
+    }
+  }
+
+  fragments.sort((a, b) => a.category > b.category ? 1 : -1);
+
+  const route = '/plugins/overview';
+  const pluginName = 'overview'
```

Comment:
Maybe call this `Overview`? Right now the name of the page in the "Overview" section is a bit off.

Maybe have the "Overview" section and call the page "All Plugins"?

(The route is fine though! Like the route - keep that...)

Reply:
Yeah I prefer that naming too. Adapted